src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,353 @@
+                        onPress={() => this.toggleAllUsers()}
+                    />
+                </View>
+                <View style={[styles.peopleRowCell, styles.flex4]}>
+                    <Text style={[styles.textStrong]}>
+                        {this.props.translate('workspace.people.assignee')}
+                    </Text>
+                </View>
+                {
+                    this.props.windowWidth >= 1100 && (
+                        <View style={[styles.peopleRowCell, styles.flex4]}>
+                            <Text style={[styles.textStrong]}>
+                                {this.props.translate('common.email')}
+                            </Text>
+                        </View>
+                    )
+                }
+                <View style={[styles.peopleRowCell, styles.flex1]} />
+            </View>
+        );
+    }
+
+    render() {
+        const data = _.chain(lodashGet(this.props, 'policy.employeeList', []))
+            .map(email => this.props.personalDetails[email])
+            .filter()
+            .value();
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={this.props.translate('common.people')}
+                    onCloseButtonPress={() => Navigation.dismissModal()}
+                />
+                <ConfirmModal
+                    danger
+                    title={this.props.translate('workspace.people.removeMembersTitle')}
+                    isVisible={this.state.isRemoveMembersConfirmModalVisible}
+                    onConfirm={() => this.removeUsers()}
+                    onCancel={this.hideConfirmModal}
+                    prompt={this.props.translate('workspace.people.removeMembersPrompt')}
+                    confirmText={this.props.translate('common.remove')}
+                    cancelText={this.props.translate('common.cancel')}
+                />
+                <ScrollView style={[styles.settingsPageBackground]} bounces={false}>
+                    <View style={styles.pageWrapper}>
+                        <View style={styles.buttonRow}>
+                            <Button
+                                success
+                                text={this.props.translate('common.invite')}
+                                onPress={() => this.inviteUser()}
+                            />
+                            <Button
+                                danger
+                                style={[styles.ml2]}
+                                isDisabled={this.state.selectedEmployees.length === 0}
+                                text={this.props.translate('common.remove')}
+                                onPress={this.askForConfirmationToRemove}
+                            />
+                        </View>
+                        <View style={[styles.w100, styles.mt4]}>
+                            {
+                                this.props.policy.employeeList && (
```

Comment:
Not sure if this needs to be conditionally rendered anymore. There should be employees or `data` should always at least be an empty array now so I think we can just render the `FlatList`.

Reply:
You are right, thanks!